xarray/backends/api.py

```diff
@@ -483,12 +483,11 @@ def close(self):
 def open_mfdataset(paths, chunks=None, concat_dim=_CONCAT_DIM_DEFAULT,
                    compat='no_conflicts', preprocess=None, engine=None,
                    lock=None, data_vars='all', coords='different',
+                   infer_order_from_coords=False,
```

Comment:
Can we leave this this out of the public function signature for now, given that it doesn't work?